include/openmc/weight_windows.h

```diff
@@ -0,0 +1,72 @@
+#ifndef OPENMC_WEIGHT_WINDOWS_H
+#define OPENMC_WEIGHT_WINDOWS_H 1
+
+#include "openmc/mesh.h"
+#include "openmc/particle.h"
+
+namespace openmc {
+
+void read_weight_window_xml(); //!< read the weight window file
```

Comment:
The comment here is not really adding much, since it is basically the same as the name of the function. Maybe you could name the specific variables it fills.